### PR TITLE
Add script to stop all product tests containers

### PR DESCRIPTION
**Pull request by kokosing**

### CODE REVIEW

File: presto-product-tests/bin/run_on_docker.sh [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+source ${BASH_SOURCE%/*}/lib.sh
```
Comment (findepi) [inline presto-product-tests/bin/run_on_docker.sh]: `"` quote this

File: presto-product-tests/bin/lib.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+source ${BASH_SOURCE%/*}/locations.sh
```
Comment (findepi) [inline presto-product-tests/bin/lib.sh]: `"` quote

File: presto-product-tests/bin/lib.sh [hunk 3]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```
Comment (findepi) [inline presto-product-tests/bin/lib.sh]: Change commit message to "Extract .." .

File: presto-product-tests/bin/stop_all_containers.sh [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
```
Comment (findepi) [inline presto-product-tests/bin/stop_all_containers.sh]: `-euo pipefail`

File: presto-product-tests/bin/run_on_docker.sh [hunk 5]
```diff
@@ -60,23 +62,6 @@ function run_product_tests() {
...
-  done
-  echo "Removing dead application-runner containers"
-  local CONTAINERS=`docker ps -aq --no-trunc --filter status=dead --filter status=exited --filter name=common_application-runner`
-  for CONTAINER in ${CONTAINERS};
-  do
-    docker rm -v "${CONTAINER}"
-  done
-}
-
 function stop_all_containers() {
```
Comment (findepi) [inline presto-product-tests/bin/run_on_docker.sh]: remnant?

File: presto-hive-hadoop2/bin/run_on_docker.sh [hunk 6]
```diff
@@ -1,4 +1,6 @@
-#!/bin/bash -ex
+#!/bin/bash
+
+set -euo pipefail -x
```
Comment (findepi) [inline presto-hive-hadoop2/bin/run_on_docker.sh]: ❤️

**Comment by kokosing** [comment]
Comments applied, added some fixes and more.

**Pull request merged**